tests/view/renderer.js

```diff
@@ -1714,6 +1715,74 @@ describe( 'Renderer', () => {
 			expect( domRoot.innerHTML ).to.equal( '<ul><li>Foo<ul><li><b>Bar</b><i>Baz</i></li></ul></li></ul>' );
 		} );
 
+		// #1439
+		it( 'should force–refresh the selection in FF after a soft break to nudge the caret (non-gecko)', () => {
```

Comment:
Shouldn't the test name be something like "it should not force-refresh the selection after a soft break in non-gecko browsers" and the other one negated?

Reply:
Also, I am not that sure if we need such a test and if we can test it properly. I mean, the test works but checking if `addRange` was called is a bit dangerous. If we ever use it in the normal selection setting process the test will start failing and we will have to remove it anyway. I would be fine with manual test only, I guess.